### DISCUSSION TITLE
comment faire une pull request

### DISCUSSION THREAD

**Discussion by theTigerFox** [community/community#138594] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

je voudrais savoir comment on fait une pull request

**Comment by Tiger-Foxx** [comment, marked as answer]
dans le menu de ton depot en haut rend toi dans l'onglet pull request

**Reply by theTigerFox** [reply]
oui mais comment ensuite

**Reply by theTigerFox** [reply]
oui mais comment ensuite

**Reply by Tiger-Foxx** [reply]
tu cliques sur new pull request

**Comment by theTigerFox** [comment]
oui mais ensuite

**Reply by Tiger-Foxx** [reply]
tu cliques sur new pull request j'ai dis

**Comment by Tiger-Foxx** [comment]
tu dois ensuite faire un new pull request et puis c'est tout tu remplis les champs
